### ISSUE TITLE
undefined local variable or method 'ajaxful_rater' under mod_rails, rails 2.3.8, ruby 1.8.7

### ISSUE THREAD

**Issue by bilditup1** [edgarjs/ajaxful-rating#46]
The title essentially says it all. mod_rails won't boot my app because of this. I thought it was because I wasn't vendorizing my gems, but having verifying that I've now done that correctly, I still get the error:

undefined local variable or method 'ajaxful_rater' under mod_rails, rails 2.3.8, ruby 1.8.7, debian 

and am now quite simply at a loss. Shutting off ajaxful-rating and taking out all references to it result in the app starting up. At the same time, this is not an issue under ruby 1.9.2/webrick, so I'm not sure what the weak pt is here.

**Comment by edgarjs** [comment]
Are you requiring the gem in your configuration block?

**Comment by bilditup1** [comment]
Yep. If it was working with one server and not the other, then yeah, the presumption should be that I am requiring the gem :)
In any case we ended up using a mongrel cluster (still rails 2.3.8) under which this is not an issue.
